zellij-utils/src/envs.rs

```diff
@@ -0,0 +1,24 @@
+/// Uniformly operates ZELLIJ* environment variables
+use anyhow::Result;
```

Comment:
Why do we need the `anyhow::Result` here?

Reply:
@a-kenji 
Ah, I just thought it would be preferable to unify and absorb all result types to `anyhow::Result` that would be clear as I did on https://github.com/zellij-org/zellij/pull/824#discussion_r741408603.

If you enact how to handle result types as shown below, I think `anyhow::Result` should be removed from this PR:

1. Use `anyhow::Result` if a function returns various error types like https://github.com/zellij-org/zellij/pull/824#discussion_r741408603
2. If a function returns only one error type, just propagate it

Reply:
Thank you for the insight, 
I think the list you suggested would be preferable,
unless there is a clear benefit to using `anyhow::Result` everywhere.

What do you think?

Reply:
@a-kenji 

> Additionally, the main function also can return `anyhow::Result` as shown [here](https://docs.rs/anyhow/1.0.44/anyhow/#details).

As I mentioned this [here](https://github.com/zellij-org/zellij/pull/824#discussion_r741408603), we can have the main function return `anyhow::Result`. This means all functions can perfectly back forward errors to main and a parent process with almost panicless using `?` operator. I personally think this is the benefit to use `anyhow::Result` everywhere. However, this takes a lot of work, and the benefit may not be worth the effort; in addition, some situations might need intentionally panicking.

To conclude, I think putting `anyhow::Result` everywhere will be preferable when the following conditions are satisfied:
1. If you want to collect all errors into the main function
2. If you want to avoid almost all panics and `process::exit`

Reply:
Oh, my bad I thought it would still work that way with a normal `Result`.
But it explicitly needs to return the `anyhow::Result`.
Thank you for that explanation again!